docs/pages/modules/module-api.mdx

```diff
@@ -133,6 +133,7 @@ AsyncFunction("asyncFunction") { (message: String) in
 
 // or
 
+// Make sure to import `Promise` class from `expo.modules.kotlin`, not from `expo.modules.core`!
```

Comment:
```suggestion
// Make sure to import `Promise` class from `expo.modules.kotlin`, not from `expo.modules.core`.
```

Let's avoid using exclamation since this is a statement. A comment is enough to put emphasis on what action user needs to take.

Reply:
I didn't want to add the import statement here because when copying the code to the place where it should be in the user's app, the import statement will be in the incorrect place. That would make that more confusing, in my opinion.

Reply:
Got it! Thanks, @lukmccall, for the explanation.